geolocator/pubspec.yaml

```
@@ -1,7 +1,8 @@
 name: geolocator
 description: Geolocation plugin for Flutter. This plugin provides a cross-platform (iOS, Android) API for generic location (GPS etc.) functions.
-version: 6.1.7
+version: 6.1.8
```

Yesterday we released version 6.1.8, so this should probably be updated to 6.1.9
```suggestion
version: 6.1.9
```